__tests__/help.ts

```diff
@@ -17,6 +22,16 @@ test('npx unimported --help', (done) => {
         --flow, -f    indicates if your code is annotated with flow types    [boolean]
         --update, -u  update the ignore-lists stored in .unimportedrc.json   [boolean]"
     `);
-    done();
-  });
+  done();
+});
+
+test('error npx unimported --help', async (done) => {
+  // wrong file name
```

Comment:
The test out would be more useful if this was actually part of the test name instead of just a comment.